docker/build_windows.dockerfile

```diff
@@ -29,13 +29,14 @@ VOLUME /kolibridist/
 # TODO(cpauya): Verify the checksums of the downloaded Python installers.
 
 CMD git clone https://github.com/learningequality/kolibri-installer-windows.git && \
-    cd kolibri-installer-windows/src && \
-    git checkout $KOLIBRI_WINDOWS_INSTALLER_VERSION && \
-    cp /kolibridist/kolibri-$KOLIBRI_VERSION*.whl . && \
-    export KOLIBRI_BUILD_VERSION=$KOLIBRI_VERSION && \
-    make && \
-    curl -sS https://www.python.org/ftp/python/3.4.3/python-3.4.3.msi --output "python-setup/python-3.4.3.msi" && \
-    curl -sS https://www.python.org/ftp/python/3.4.3/python-3.4.3.amd64.msi --output "python-setup/python-3.4.3.amd64.msi" && \
-    wine inno-compiler/ISCC.exe installer-source/KolibriSetupScript.iss && \
-    mv *.exe kolibri-$KOLIBRI_VERSION-unsigned.exe && \
-    cp *.exe /kolibridist/
+      cd kolibri-installer-windows && \
+      git checkout $KOLIBRI_WINDOWS_INSTALLER_VERSION && \
+      cd src && \
```

Comment:
Thanks for the quick-fix!  Since v1.3.3 - we changed from `windows/` to `src/` directory -- see the directory structure differences in the following:

* https://github.com/learningequality/kolibri-installer-windows/tree/v1.3.3
* https://github.com/learningequality/kolibri-installer-windows/tree/v1.3.2

To ensure backwards-compatibility - let us check for either the `windows/` or `src/` directory if it exists before doing a `cd` to it?  

So if the environment variable `KOLIBRI_WINDOWS_INSTALLER_VERSION` is set with `v1.3.2` (or older versions) - this script won't throw an exception.